pkg/controller/volume/persistentvolume/testing/testing.go

```diff
@@ -219,21 +221,19 @@ func (r *VolumeReactor) React(action core.Action) (handled bool, ret runtime.Obj
 		if found {
 			klog.V(4).Infof("GetVolume: found %s", volume.Name)
 			return true, volume.DeepCopy(), nil
-		} else {
-			klog.V(4).Infof("GetVolume: volume %s not found", name)
-			return true, nil, fmt.Errorf("Cannot find volume %s", name)
 		}
+		klog.V(4).Infof("GetVolume: volume %s not found", name)
```

Comment:
`if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)`
it applies to all `if else` cases